src/containers/Home/index.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
+import useForm, { isRequired, isEmail, maxChars } from '../../hooks/useForm';
```

Comment:
MINOR: Yeah definitely isEmail and macChars should be a different module.

Reply:
agree